frontend/src/features/map_planning/layers/plant/hooks/useFindPlantFromSeedCallback.ts

```diff
@@ -16,8 +15,12 @@ export function useFindPlantFromSeedCallback(afterPlantLoad: (plant: PlantForPla
   const { t } = useTranslation(['plantings']);
   const [plantId, setPlantId] = useState(0);
   const [seed, setSeed] = useState<SeedDto | null>(null);
-  const { data: plant, error } = useQuery(['plants/plant', plantId] as const, {
+  const { data: plant } = useQuery(['plants/plant', plantId] as const, {
     queryFn: (context) => findPlantById(context.queryKey[1]),
+    meta: {
+      autoClose: false,
+      errorMessage: plantId !== 0 ? t('plantings:error_fetching_plant') : undefined,
```

Comment:
What happens if errorMessage is undefined?

Reply:
Is the error suppressed and is this wanted?

Reply:
if errorMessage is undefined than no toast is shown:

```
const queryClient = new QueryClient({
  queryCache: new QueryCache({
    onError: (error, query) => {
      if (query.meta?.errorMessage && typeof query.meta.errorMessage === 'string') {
        errorToastGrouped(query.meta.errorMessage);
      }
    },
  }),
});
```

i just refactored the existing logic. Before refactoring it was also checked if plant id !== 0

```
if (error && plantId !== 0) {
    errorToastGrouped(t('plantings:error_fetching_plant'), { autoClose: false });
}
```